torch/_dynamo/utils.py

```diff
@@ -255,21 +256,38 @@ def add(self, key):
 graph_break_dup_warning_checker = DuplicateWarningChecker()
 
 
-def init_logging():
-    torchdynamo_logging.init_logging(
-        config.log_level, log_file_name=config.log_file_name
-    )
-    graph_break_dup_warning_checker.reset()
+def setup_compile_debug():
```

Comment:
Wouldn't it make more sense for logging initialization to take care of this directly? Otherwise, `TORCH_COMPILE_DEBUG` will do nothing if you don't actually call `setup_compile_debug`. Also, you can use this to warn if a user is inconsistently using torch compile bug along with the default envvars.

Reply:
So the reason I didn't do this was to keep _init_logs agnostic, which I think is good for maintenance, this will also be useful for the issue where we want to add file handlers to the artifacts, because the artifact logs will have been initialized by the time we are running.

I consider this "custom logging behavior for the component" and expect this to be the pattern people use rather than having more logic in the relatively lightweight _logging module.